.travis.yml

```diff
@@ -64,6 +64,13 @@ install:
   - pip install google-cloud
   - pip install Pillow
 
+
+before_script:
+    # stop the build if there are Python syntax errors or undefined names
+    if [[ "$TENSORFLOW_V" == "1.8.0" ]]; then
```

Comment:
is there another way to trigger this test? I'm concerned that if we change the versions of tensorflow we use, this will silently stop running and we won't notice

Reply:
If you have a better solution, I am all ears.  I did not want flake8 to run 4 times but I did want it to run on both Py2 and Py3.  I figured that 1.8.0 was the latest version under test so it would change last.

Reply:
What about making an explicit `RUN_FLAKE8` variable and setting it in the `matrix` section, with a comment that it should be on for exactly one job each for Python 2 and Python 3? Long term maintainers will read comments but probably won't read commit messages, especially considering that the original commit may not show up in the blame layer after a while.